src/emc/task/taskmodule.cc

```diff
@@ -330,12 +325,10 @@ BOOST_PYTHON_MODULE(emctask) {
 	;
 
 #pragma GCC diagnostic push
```

Comment:
can we be rid of the diagnostic push and (below) pop too? They exist only to avoid a compoiler diagnostic when the deprecated field was referred to.